.ci_support/osx_64_.yaml

```diff
@@ -19,7 +19,7 @@ cxx_compiler:
 cxx_compiler_version:
 - '16'
 flatbuffers:
-- 24.3.6
+- 23.5.26
```

Comment:
I take it things are not compatible with flatbuffers 24.3?

Reply:
such version is not available
![image](https://github.com/conda-forge/openvino-feedstock/assets/2566854/e1c71f26-a131-4304-9024-5c9a6a26a7c1)

Reply:
lets see... https://github.com/conda-forge/flatbuffers-feedstock/issues/50